docs/how-to-guides/basic_image.rst

```diff
@@ -0,0 +1,99 @@
+.. _howto_buld_basic_image:
+
+===================
+Build a basic image
+===================
+
+These instructions describe how to build a ``classic`` server 24.04 AMD64 Ubuntu image
+with Imagecraft.
+
+Prerequisites
+-------------
+
+- AMD64 machine with Ubuntu 24.04
+- ``snapd`` installed
+- 15GiB or more disk space to process the build and hold the resulting image
+
+.. note:: Following instructions build an **AMD64** image on an **AMD64** machine.
+          Building on another architecture would need several modifications not
+          described on this page.
+
+Install Imagecraft
+~~~~~~~~~~~~~~~~~~
+
+Imagecraft is only available as a snap in the ``latest/edge`` channel
+`from the Snapstore <https://snapcraft.io/imagecraft>`_. Install it with:
+
+.. code-block::
+
+    sudo snap install --classic --edge imagecraft
```

Comment:
```suggestion
    sudo snap install imagecraft --channel=edge --classic
```
A minor difference, but this pattern more closely aligns with what guidance in Snacpraft.